scalameta/inputs/shared/src/main/scala/scala/meta/inputs/Input.scala

```diff
@@ -19,9 +19,9 @@ object Input {
     override def toString = "Input.None"
   }
 
-  @leaf class String(s: scala.Predef.String) extends Input {
-    lazy val chars = s.toArray
-    override def toString = s"""Input.String("$s")"""
+  @leaf class String(value: scala.Predef.String) extends Input {
```

Comment:
Can we document these breaking changes in a markdown file? It will make it a lot easier to implement migration rewrites in scalafix than me digging through the commit history.